bin/daru-view

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'rake'
+
+path = `gem which daru/view`.split('/')[0...-1].join('/')
+
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: daru-view [options]"
+  # Update google charts javascript dependent files, from latest Builds on google developers website
+  opts.on('-g', '--update-googlecharts', 'Update googlecharts.js libraries') do
+    sh "curl -# http://www.google.com/jsapi -L --compressed -o #{path}/view/adapters/js/googlecharts_js/google_visualr.js"
```

Comment:
It must create folder and file if it is not present, right ?

Reply:
Yes, will make that change.

Reply:
What if file is also not present or deleted to update it ? `touch` can be useful here, right ?

Reply:
Even I tried deleting the files. But it was throwing error. I hope you can write testcases for it. Please write in comment the TODO part as well.

Reply:
Updated the travis script to run the executable and added the TODO.